src/resources/views/crud/list.blade.php

```diff
@@ -159,6 +159,13 @@ class="{{ backpack_theme_config('classes.table') ?? 'table table-striped table-h
   @basset('https://cdn.datatables.net/fixedheader/3.3.1/css/fixedHeader.dataTables.min.css')
   @basset('https://cdn.datatables.net/responsive/2.4.0/css/responsive.dataTables.min.css')
 
+  <style>
+      /* Use whole table width for td when displaying empty content message */
+      td.dataTables_empty {
+          display: table-cell !important;
+      }
+  </style>
+
```

Comment:
Aaah.... this style block with just 1 direction makes me think 🚩

😀 Isn't there a better place we can put this style? Don't we have a CSS file in CRUD, or something? Where did we put other dataTable CSS customizations?

Reply:
Had the same thought... I should have found a better place for it.

Let me recheck it ☺️